examples/new_model.py

```diff
@@ -111,6 +111,9 @@ def future_load(t, x=None):
     print('\nThis event that occured first: ', event_occured)
     # It falls before it hits the gorund, obviously
 
+    # Metrics can be analyzed from the simulation results.
+    print('\nMonotonicity: ', simulated_results.states.monotonicity())
```

Comment:
```suggestion
    print('\nMonotonicity: ', simulated_results.event_states.monotonicity())
```

Monotonicity makes more sense for event states